pandas/tests/io/formats/style/test_style.py

```diff
@@ -1258,21 +1257,6 @@ def test_column_and_row_styling(self):
         )
         assert "#T__ .row0 {\n  color: blue;\n}" in s.render()
 
-    def test_colspan_w3(self):
```

Comment:
these are just moved?

Reply:
yes just moved to `test_html.py` since HTML related.